jobs/integration/conftest.py

```diff
@@ -150,13 +150,20 @@ async def run(self, cmd, *args, stdin=None):
             )
         return stdout.decode("utf8"), stderr.decode("utf8")
 
-    async def juju_wait(self, *args, **kwargs):
-        cmd = ["/snap/bin/juju-wait", "-e", self.connection, "-w", "-v"]
-        if args:
-            cmd.extend(args)
-        if "timeout_secs" in kwargs and kwargs["timeout_secs"]:
-            cmd.extend(["-t", str(kwargs["timeout_secs"])])
-        return await self.run(*cmd)
+    def juju_wait(self, *args, **kwargs):
+        """Run juju-wait command with provided arguments.
+
+        if kwarg contains `m`: juju-wait is executed on a different model
+        see juju-wait --help for other supported arguments
+        """
+
+        command = sh.Command("/snap/bin/juju-wait")
+        if "m" not in kwargs:
+            kwargs["m"] = self.connection
+        debug = partial(click.echo, nl=False)
+        result = command("-w", "-v", *args, **kwargs, _err=debug, _tee="err")
```

Comment:
tee`ing the stderr from `juju-wait` should provide realtime results from juju-wait as well as showing the response from juju-wait in the ci.log file